components/footer.tsx

```diff
@@ -1,35 +1,133 @@
 export default function Footer() {
+    const navigation = {
+        solutions: [
+            { name: "Marketing", href: "#" },
+            { name: "Analytics", href: "#" },
+            { name: "Commerce", href: "#" },
+            { name: "Insights", href: "#" }
+        ],
+        support: [
+            { name: "Pricing", href: "#" },
+            { name: "Documentation", href: "#" },
+            { name: "Guides", href: "#" },
+            { name: "API Status", href: "#" }
+        ],
+        company: [
+            { name: "About", href: "#" },
+            { name: "Blog", href: "#" },
+            { name: "Jobs", href: "#" },
+            { name: "Press", href: "#" },
+            { name: "Partners", href: "#" }
+        ],
+        legal: [
+            { name: "Claim", href: "#" },
+            { name: "Privacy", href: "#" },
+            { name: "Terms", href: "#" }
+        ],
+        social: [
+            {
+                name: "X",
+                href: "https://x.com/Bitcoin_Devs",
+                icon: (props: any) => (
+                    <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
```

Comment:
instead of embedding the svgs in the component file, you can add them to the public folder and import them into this file to use in a next image tag. That way the code is modular, less bloated and svgs are reusable else where